### PR TITLE
Updating channels happens on uris now

### PR DESCRIPTION
**Pull request by Spartakovic**
to ensure that there will be no duplicated channels in db

### CODE REVIEW

File: src/test/java/org/atlasapi/media/channel/MongoChannelStoreWriteTest.java [hunk 1]
```diff
@@ -2,24 +2,35 @@
...
 import com.google.common.collect.ImmutableSet;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
 import com.metabroadcast.common.base.Maybe;
 import com.metabroadcast.common.persistence.MongoTestHelper;
 import com.metabroadcast.common.persistence.mongo.DatabasedMongo;
 
+@RunWith(MockitoJUnitRunner.class)
```
Comment (Dalamar42) [inline src/test/java/org/atlasapi/media/channel/MongoChannelStoreWriteTest.java]: Why do you need the mockito runner? I don't see any mocks

File: src/test/java/org/atlasapi/media/channel/MongoChannelStoreWriteTest.java [hunk 2]
```diff
@@ -2,24 +2,35 @@
...
 public class MongoChannelStoreWriteTest extends TestCase {
```
Comment (Dalamar42) [inline src/test/java/org/atlasapi/media/channel/MongoChannelStoreWriteTest.java]: I presume there are other tests here that ensure writing/updating channels works?

**Comment by Dalamar42** [comment]
Assuming we have existing tests making sure we can create/update channels LGTM

File: src/test/java/org/atlasapi/media/channel/MongoChannelStoreWriteTest.java (hunk 1 above)
Comment (Spartakovic) [inline src/test/java/org/atlasapi/media/channel/MongoChannelStoreWriteTest.java]: That what I thought, but apparently Rule that checks exceptions also needs that

**Pull request merged**